### DISCUSSION TITLE
elastalert-create-index: not found error when running ElastAlert2 as a docker container.

### DISCUSSION THREAD

**Discussion by JackwinJoseph3121** [jertel/elastalert2#1431] [Q&A] [answered]
I am trying to run elastalert2 as a docker container and passing the configuration files and rules folder as a bind mount. Here is the command below 
`docker run --net=es_default -d --name elastalert --restart=always --mount type=bind,source=/elastalert/config/config.yaml,target=/opt/elastalert/config.yaml --mount type=bind,source=/elastalert/rules,target=/opt/elastalert/rules --mount type=bind,source=/elastalert/bin,target=/usr/local/bin ghcr.io/jertel/elastalert2/elastalert2  --verbose`
But the docker container is not starting. When we check the docker logs it shows :`/opt/elastalert/run.sh: 3: elastalert-create-index: not found`.
The config,yaml are as follows:

`es_host: eshost
es_port: 9200
#es_username:
es_api_key: "api_key"
verify_certs: False
use_ssl: True
ssl_show_warn: True
writeback_index: elastalert_status
writeback_alias: elastalert_alerts
alert_time_limit:
  days: 2

rules_folder: ./rules

run_every:
  minutes: 5

buffer_time:
  minutes: 15
`

**Comment by jertel** [comment, marked as answer]
The [documentation](https://elastalert2.readthedocs.io/en/latest/running_elastalert.html#as-a-docker-container) describes how to start ElastAlert2 via Docker.

**Reply by JackwinJoseph3121** [reply]
We ran the same configuration in our test environment as a python package and the configuration seems to be working fine. We wanted to see if the same applies for running as a docker container. We are not able to pass elastalert-create-index to manually create an index. We didn't see much information based on it. We just wanted to know if we can do these configurations with docker. Thanks for your timely reply btw.
